src/sentry/static/sentry/app/components/clippedBox.jsx

```diff
@@ -48,18 +51,24 @@ const ClippedBox = React.createClass({
     if (this.state.clipped) {
       className += ' clipped';
     }
+    if (this.state.revealed) {
+      className += ' revealed';
+    }
 
     return (
       <div className={className}>
         {this.props.title &&
           <h5>{this.props.title}</h5>
         }
         {this.props.children}
-        <div className="clip-fade">
-          <a onClick={this.reveal} className="show-more btn btn-primary btn-xs">
-            {t('Show more')}
-          </a>
-        </div>
+
+        {this.state.clipped &&
```

Comment:
Small optimization – it's better not to inject this DOM node if we don't need it